processing file asmFiles/beq_notTaken.asm (1 of 24)
processing file asmFiles/beq_taken.asm (2 of 24)
processing file asmFiles/bge_notTaken.asm (3 of 24)
processing file asmFiles/bge_taken.asm (4 of 24)
processing file asmFiles/bgeu_notTaken.asm (5 of 24)
processing file asmFiles/bgeu_taken.asm (6 of 24)
processing file asmFiles/blt_notTaken.asm (7 of 24)
processing file asmFiles/blt_taken.asm (8 of 24)
processing file asmFiles/bltu_notTaken.asm (9 of 24)
processing file asmFiles/bltu.taken.asm (10 of 24)
processing file asmFiles/bne_notTaken.asm (11 of 24)
processing file asmFiles/bne_taken.asm (12 of 24)
processing file asmFiles/count_days.asm (13 of 24)
processing file asmFiles/fib.asm (14 of 24)
processing file asmFiles/halt.asm (15 of 24)
processing file asmFiles/jal.asm (16 of 24)
processing file asmFiles/jalr.asm (17 of 24)
--- memsim.hex	2026-09-13 16:57:20.770472413 -0400
+++ memcpu.hex	2026-09-13 16:58:03.322048587 -0400
@@ -1,11 +1 @@
-:0400000000000001FB
-:040001000030202388
-:040002000080026F09
-:04000300FFFFFFFFFD
-:0400040000C00293A3
-:04000500005204633E
-:04000600FFFFFFFFFA
-:040007000010019351
-:040008000030202381
-:04000900FFFFFFFFF7
 :00000001FF
processing file asmFiles/mergesort.asm (18 of 24)
processing file asmFiles/mult.asm (19 of 24)
processing file asmFiles/mult_procedure.asm (20 of 24)
processing file asmFiles/search.asm (21 of 24)
processing file asmFiles/test.branchjump1.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
